src/Pagination.js

```diff
@@ -0,0 +1,240 @@
+'use strict';
+
+import core from 'metal';
+import dom from 'metal-dom';
+import templates from './Pagination.soy';
+import Component from 'metal-component';
+import Soy from 'metal-soy';
+
+class Pagination extends Component  {
+	constructor(value) {
```

Comment:
It's best to use the `created` lifecycle function instead of `constructor` here, since `created` is guaranteed to run before the first render, and doesn't require calling the `super` method, so you won't risk forgetting to pass some param to it (the super constructor for instance can receive two params, so this wouldn't work well if someone passed the second param to your component).

Example: https://github.com/metal/metal-tooltip/blob/master/src/TooltipBase.js#L27